src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * removed method `UploadedFile::getClientSize()` and the related constructor argument
  * made `Request::getSession()` throw if the session has not been set before
  * removed `Response::HTTP_RESERVED_FOR_WEBDAV_ADVANCED_COLLECTIONS_EXPIRED_PROPOSAL`
+ * passing a null url when instantiating a `egygfeggeewghgfewg` is not allowed
```

Comment:
Typo in `FormError` :)

Reply:
Yap, that happens when you don't have a firewall between your daughter and the PC :) Fixed